app/models/admin_set.rb

```diff
@@ -1,5 +1,7 @@
 # There is an interplay between an AdminSet and a PermissionTemplate.
 # @see Hyrax::AdminSetBehavior for further discussion
 class AdminSet < ActiveFedora::Base
+  DEFAULT_ID = 'admin_set/default'.freeze
```

Comment:
What about exposing a configuration option in the engine? If there is a chance we are overriding, I'd prefer exposing a defined configuration method instead of overriding a constant (which feels very questionable given that a constant implies unchanging).